pkg/router/routeapihelpers/validation_test.go

```diff
@@ -928,49 +932,283 @@ IrRGZJwgzmWX+NzqK9H3AyFk5p9oBuzmulVoJyKFzs1eN4ZIn25ifP8hP+uJHOTE
 jZrtwVw4rGVb/qM=
 -----END PRIVATE KEY-----`
 
-	// openssl req -x509 -sha1 -newkey rsa:1024 -days 3650 -keyout exampleca.key -out exampleca.crt -addext "keyUsage=cRLSign, digitalSignature, keyCertSign" -addext "extendedKeyUsage=serverAuth, clientAuth" -nodes -subj '/C=US/ST=SC/L=Default City/O=Default Company Ltd/OU=Test CA/CN=www.exampleca.com/emailAddress=example@example.com'
+	// openssl req -x509 -sha1 -newkey rsa:1024 -days 3650 -keyout testCertificateRsaSha1CA.key -out testCertificateRsaSha1CA.crt -addext "keyUsage=cRLSign, digitalSignature, keyCertSign" -addext "extendedKeyUsage=serverAuth, clientAuth" -nodes -subj '/C=US/ST=SC/L=Default City/O=Default Company Ltd/OU=Test CA/CN=www.exampleca.com/emailAddress=example@example.com'
+	//
+	// Key = testCertificateRsaSha1CAKey
+	// CA = self-signed
+	testCertificateRsaSha1CA = `-----BEGIN CERTIFICATE-----
```

Comment:
nit on variable name:
```suggestion
	testCertificateRsaSha1SelfSignedCA = `-----BEGIN CERTIFICATE-----
```

Reply:
I see you use the name already below. Whatever you can use that identifies this as self-signed would help.  Otherwise I feel the need to see godoc of cert variable names to their primary characteristics.  I know that's asking alot.